hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/WorldLedgers.java

```diff
@@ -78,6 +79,7 @@ public class WorldLedgers {
 
     private final ContractAliases aliases;
     private final StaticEntityAccess staticEntityAccess;
+    private final FcmCustomFeeSchedules customFeeSchedules;
```

Comment:
Since most of the time we won't need this field, and it can be easily constructed on-request, imo we should delete this field and just do,
```
    public FcmCustomFeeSchedules customFeeSchedules() {
        return new FcmCustomFeeSchedules(tokensLedger);
    }
```

Reply:
DONE